lc-hax/Scripts/Commands/MaskCommand.cs

```diff
@@ -3,22 +3,43 @@
 
 [Command("/mask")]
 public class MaskCommand : ICommand {
-    public void Execute(StringArray args) {
-        if (args.Length is 0) {
-            Chat.Print("Usage: /mask <player>");
-            return;
-        }
 
-        if (Helper.GetActivePlayer(args[0]) is not PlayerControllerB targetPlayer) {
-            Chat.Print("Player not found!");
+    public void Execute(StringArray args) {
+        if (args.Length == 0) {
+            Chat.Print("Usage: /mask self OR /mask <player> (optional) <amount>");
             return;
         }
 
         if (Helper.LocalPlayer?.currentlyHeldObjectServer is not HauntedMaskItem hauntedMaskItem) {
             Chat.Print("You are not holding a mask!");
             return;
         }
+        PlayerControllerB targetPlayer;
+
+        if (args[0].ToLower() == "self") {
+            targetPlayer = Helper.LocalPlayer;
+        }
+        else {
+            if (Helper.GetActivePlayer(args[0]) is not PlayerControllerB player) {
+                Chat.Print("Player not found!");
+                return;
+            }
+            targetPlayer = player;
+        }
```

Comment:
I really dislike using `self` as an argument.

1. This is not consistent behaviour with other commands. Instead passing `/mask` without arguments can short-circuit to summon a mask to yourself by default.
2. I don't see summoning a mask on yourself to be a common occurrence and on the rare case they do, they can just write their own username or player ID.

Reply:
self can be a alias for the local player, which makes it fast, but you can make your own choice.

Reply:
I understand, but we have other commands that operate similarly. For example `/kill` is an alias for `/kill <your username>`. This is not consistent behaviour.

Reply:
Do any changes you feel happy with, i cannot figure how to make /mask <number> and act it work on without self param

Reply:
Sure but I am considering to not support the alias. Is there a good reason why you need to use `/mask` on yourself frequently?

Reply:
Well, i would love to spawn maskeds on certain locations in phantom, but not with them being so close to a player, so i hide somewhere, spawn a bunch of maskeds when i want to troll in a confined area, then get far and open the door so they blend well and surprise everyone

Reply:
This is a great reason. I'll support it.

Reply:
> This is a great reason. I'll support it.

I might have a idea how to force the maskeds custom location and change the suit as well using harmony, ill figure something out